tests/slow/SwizzledTenantManagement.toml

```diff
@@ -1,6 +1,7 @@
 [configuration]
 allowDefaultTenant = false
 tenantModes = ['optional', 'required']
+disableEncryption = true
```

Comment:
Can we just disable storage server encryption instead of encryption altogether. For example there are alot of these workloads which test encrypted snapshot backup up so I would prefer to keep them running

Reply:
The TenantManagement tests don't test snapshot backup. For the AtomicRestore tests and the UpgradeAndBackupRestore tests let me see if I can change them to also restore to remote cluster while keeping validation workloads running.